.github/workflows/checks.yml

```diff
@@ -44,6 +44,6 @@ jobs:
     - name: isort
       run: isort . --check --diff
     - name: mypy
-      run: mypy .
+      run: mypy --follow-imports=normal .
```

Comment:
In my environment, the following does not import `torch` as intended:

```shell
rm -r .mypy_cache/; time mypy optuna tests --follow-imports=normal
```

Reply:
@eukaryo Could you confirm it?

Reply:
I realized that I had misunderstood about precedence order of configurations.
`--follow-imports=normal` in command-line option cannot override `[mypy-torch.*]` section in the config file.
(cf. https://mypy.readthedocs.io/en/stable/config_file.html )

Reply:
I have revised the code.
`rm -r .mypy_cache/; awk '/\[mypy-torch\.\*\]/{exit} {print}' setup.cfg | (time mypy --config-file=/dev/stdin .)`
This command will work as intended. Could you confirm it?